OpenTabletDriver.UX/MainForm.cs

```diff
@@ -530,8 +540,34 @@ private async Task SaveSettings()
             }
         }
 
+        private CancellationTokenSource _disableApplySaveButtons = new();
+
+        private void DisableApplySaveButtons(bool disableSave = true)
+        {
+            Application.Instance.InvokeAsync(async () =>
+            {
+                await _disableApplySaveButtons.CancelAsync();
+                _disableApplySaveButtons = new CancellationTokenSource();
+                if (disableSave)
+                    saveButton.Enabled = false;
+
+                applyButton.Enabled = false;
+
+                // debounce re-enabling task
+                await Task.Delay(2500, _disableApplySaveButtons.Token).ContinueWith(task =>
```

Comment:
```suggestion
                await Task.Delay(1000, _disableApplySaveButtons.Token).ContinueWith(task =>
```

2.5s might be too much here. The goal here is mainly to give the user feedback that their click actually went through. I feel like I'm watching paint dry waiting 2.5s for the button to enable itself again and I could totally see someone making some tiny mistake in their settings and being annoyed at not being able to very quickly apply/save again.